docs/jah/ack/index.md

```diff
@@ -0,0 +1,123 @@
+---
+---
+
+# Ack
+
+Simple sample player
+
+## Features
+
+- 8 voice sample playback of mono or stereo samples
+- Sample start / end and loop positions
+- Resonant multimode filter
+- Envelopes for volume and filter cutoff
+- Master delay and reverb
+
+## Commands
+
+_Note: channels are indexed 0-7 in commands._
```

Comment:
aside: curious why not 1-8?

Reply:
good question. i dunno :) other than the fact that sc typically start indices with 0 and lua with 1.

Reply:
also, looking at other monome osc protocols (serialosc) indices start with 0.

Reply:
👍 

lua's outsider status strikes again!  (where's the `:punk:` emoji?)

anyway, something to consider making crystal clear in the general docs (fyi @tehn)

Reply:
fwiw, in other engines that take index arguments, i've been keeping them 1-based and shifting them at the receiving end (in sclang)

Reply:
oh. cool!  i wonder if we shouldn't just adopt that as a convention?

Reply:
yeah if we establish best practices / naming standards on engines i'll change to that. right now i do the shift in the generic `dust/lib/jah/ack.lua` module.